docs/howto/manage-charms.md

```diff
@@ -503,7 +498,7 @@ Updating a charm to the latest revision always involves the `refresh` command, b
 ### Update a Charmhub charm
 
 ```{important}
-Because of the way charm channels work, 'updating' doesn't have to mean 'upgrading' -- you can switch to any charm revision, no matter if it's newer or older. The instructions below reflect this.
+Because of the way charm {ref}`channels <charm-channel>` work, 'updating' doesn't have to mean 'upgrading' -- you can switch to any charm revision, no matter if it's newer or older. The instructions below reflect this.
 
 However, as newer versions typically contain improvements, Juju will notify you if a new version exists: Juju polls Charmhub once a day to check for updates and, if an update is found, the poll will cause `juju status` to indicate that a newer charm version is available.
```

Comment:
you have to use `juju status --format json|yaml` to see that data.

Reply:
That's not true, also shows in tabular
```		w.Print(app.CharmName, app.CharmChannel)
		if app.CanUpgradeTo != "" {
			w.PrintColor(output.WarningHighlight, app.CharmRev)
		} else {
			w.Print(app.CharmRev)
		}
```